docs/bucket/replication/README.md

```diff
@@ -66,8 +66,8 @@ Replication configuration applied successfully to myminio/srcbucket.
 
 The replication configuration follows [AWS S3 Spec](https://docs.aws.amazon.com/AmazonS3/latest/dev/replication-add-config.html). Any objects uploaded to the source bucket that meet replication criteria will now be automatically replicated by the MinIO server to the remote destination bucket. Replication can be disabled at any time by disabling specific rules in the configuration or deleting the replication configuration entirely.
 
-
-When an object is deleted from the source bucket, the replica will not be deleted as per S3 spec.
+When an object is deleted from the source bucket, the replica will be deleted if delete marker replication is enabled in the replication configuration.By default, if delete marker replication is enabled - MinIO performs active-active replication
```

Comment:
```suggestion
When an object is deleted from the source bucket, the replica will be deleted if delete marker replication is enabled in the replication configuration. By default, if delete marker replication is enabled - MinIO performs active-active replication
```

Reply:
This needs rewording "By default, if delete marker replication is enabled - MinIO performs active-active replication". Delete marker replication does not imply that 2 buckets are replicating to each other?

Reply:
reworded.